doc/engineering-reference/src/simulation-models-encyclopedic-reference-002/demand-controlled-ventilation.tex

```diff
@@ -290,18 +290,31 @@ \subsection{Proportional Control}\label{proportional-control}
 {V_{ot - design,i}} = \frac{{\left( {{R_{p,i}}.{P_{z,i}} \times {SchedFrac_i}} \right) + ({R_{a,i}}.{A_{z,i}} \times {SchedFrac_i})}}{{{E_i}}}
 \end{equation}
 
+if ProportionalControlBasedOnDesignOARate is specified, the required intake flow rate will be revised as:
+
+ \begin{equation} {V{ot - design,i}} = \frac{Design Outdoor Air Flow Rate_i} {E_i} \end{equation}
+
+The Design Outdoor Air Flow Rate is determined in the DesignSpecification:OutdoorAir object. 
+
 2)~~~The required intake flow of outdoor air when the zone is unoccupied i.e.~ \emph{P\(_{z,i}\)} = 0
 
 \begin{equation}
 {V_{ot - min,i}} = \frac{{({R_{a,i}}.{A_{z,i}} \times {SchedFrac_i})}}{{{E_i}}}
 \end{equation}
 
+if ProportionalControlBasedOnDesignOARate is specified, the required intake flow rate will be revised as:
+
+ \begin{equation} {V{ot - min,i}} = \frac{ {Design Outdoor Air Flow Rate_i} \times {SchedPropMin_i} } {E_i} \end{equation}
+
+
 3)~~~The target indoor CO\(_{2}\) concentration at \({V_{ot - design,i}}\)
 
 \begin{equation}
 {C_{s - design,i}} = {C_o} + \frac{N}{{\left( {{V_{ot - design,i}}/{P_{z,i}}} \right)}}
 \end{equation}
 
+When the field of Maximum Carbon Dioxide Concentration Schedule Name is entered, the schedule value is set to {C_{s - design,i}}. 
```

Comment:
I'm still not sure I understand the wording here.  You are saying:

> When the field of Maximum Carbon Dioxide Concentration Schedule Name is entered

which makes me think the user is entering a schedule of known values they want to use.

But then you say:

> the schedule value is set to {C_{s - design,i}}

which makes me think you are overriding their schedule value with this newly calculated value of C_s_design.

I think you are saying "When the schedule is entered, the schedule value will be used, overriding the calculated value".  Is that right?  I'm going to assume it's right and make that change here.